themes/OpenFisca-2022/layouts/partials/card.html

```diff
@@ -5,12 +5,15 @@
 	{{ $imageSrc = $newImageSrc}}
 {{ end }}
 
-<div class="card {{ if .website }}card-hasLink{{ end }}">
-	<div class="card_image">
-		<img class="screenshot" src="{{ $imageSrc }}" alt="{{ .title }}" />
-	</div>
+<div class="card{{ if .website }} card-hasLink{{ end }}{{ with .until }} card-isUntil{{ end }}">
+	{{ if not .until }}
+		<div class="card_image">
+			<img class="screenshot" src="{{ $imageSrc }}" alt="{{ .title }}" />
+		</div>
+	{{ end }}
 	<div class="card_titleWrapper">
-		<h3 class="card_title">{{ .title }}</h3>
+		{{ if not .until }}<h3 class="card_title">{{ .title }}</h3>{{ end }}
+		{{ if .until }}<h4 class="card_title">{{ .title }}</h4>{{ end }}
```

Comment:
> I believe the code would be simpler and thus more maintainable by using CSS to style instead of changing the HTML.

Reply:
I understand, but in view of the low level of additional complexity, I am in favour of changing the level of the title and not just its style to be semantically more accurate.